### ISSUE TITLE
S2 bottom pattern hit map not used in fax & s2_mean_area_fraction_top not compatible with the map used

### ISSUE THREAD

**Issue by mcfatelin** [XENON1T/pax#491]
After a more close look at code:

- I found that currently the S2 bottom pattern hit map is not used in simulating the S2 hits. See [L637 of pax.simulation](https://github.com/XENON1T/pax/blob/master/pax/simulation.py#L637). The bottom S2 pattern is assumed to be uniform.
- In the mean time, fax uses a constant value of s2-top-fraction to allocate photons for a simulated S2 on top/bottom PMTs (See [L245 of XENON1T.ini](https://github.com/XENON1T/pax/blob/master/pax/config/XENON1T.ini#L245) ). And the value is not updated to the latest value got from new MC simulation (See [here](https://github.com/XENON1T/XeAnalysisScripts/blob/master/mapmaking/S2%20LCE%20mapmaking.ipynb) ). This is the reason we see discrepancy of AFT between data and MC in [Joey's note Fig.3 ](https://xecluster.lngs.infn.it/dokuwiki/lib/exe/fetch.php?media=xenon:xenon1t:sim:notes:jhowlett:preliminary_fax_only_studies_170131.html)

Suggest to:

- Update the s2_mean_area_fraction_top of [L245 of XENON1T.ini](https://github.com/XENON1T/pax/blob/master/pax/config/XENON1T.ini#L245) to the latest value from [MC simulation](https://github.com/XENON1T/XeAnalysisScripts/blob/master/mapmaking/S2%20LCE%20mapmaking.ipynb)
- Or in long term to include also bottom PMTs in S2 hit pattern map so that we get rid of the separate treat of S2 hit on top/bottom PMTs. [L637 of pax.simulation](https://github.com/XENON1T/pax/blob/master/pax/simulation.py#L637)

**Comment by JelleAalbers** [comment]
Yes, this is correct. I agree changing the value in the ini would be the easy solution for now, including the bottom PMT hit map would be the best solution long-term.

**Comment by JelleAalbers** [comment]
I'll change the title since I understand you've resolved the second part of the issue. Once a top+bottom S2 map is made, we'll still have to change fax to support it (and still support top-only maps ideally).

**Comment by l-althueser** [comment]
Fyi: The latest S2 map is extracted from MC for top+bottom PMTs. Changes in PAX/FAX are still missing.
